libweb3jsonrpc/WebThreeStubServerBase.cpp

```diff
@@ -194,7 +194,16 @@ static dev::eth::LogFilter toLogFilter(Json::Value const& _json)	// commented to
 	}
 	if (!_json["topics"].empty())
 		for (unsigned i = 0; i < _json["topics"].size(); i++)
-			filter.topic(i, jsToFixed<32>(_json["topics"][i].asString()));
+		{
```

Comment:
I added bracers to avoid dangling else. That's Xcode suggestion with which I agree. It's just more readable.

![screen shot 2015-04-24 at 15 18 35](https://cloud.githubusercontent.com/assets/2613714/7319373/52323abc-ea95-11e4-8433-bc982e1c0c72.png)